### PR TITLE
Deprecate PPP_CONFIG_DIR for specifying config path

### PR DESCRIPTION
**Pull request by djhoese**
This is a continuation of #169 to more fully deprecate PPP_CONFIG_DIR.

This may need updating to have it raise an exception completely. Further discussion is needed. Due to the previous deprecation warning, this PR should not be released until 1.9 is ready.

CC @pnuu @mraspaud @adybbroe 

 - [ ] Closes #xxxx 
 - [ ] Tests added 
 - [ ] Tests passed 
 - [ ] Passes ``flake8 pyorbital`` 
 - [ ] Fully documented

### CODE REVIEW

**Comment by djhoese** [comment]
Oh I also made pytest more strict and removed usage of datetime utcnow.

**Review by adybbroe — APPROVED** [review]
Looks good, thanks.

**Comment by adybbroe** [comment]
> I don't see `utcnow` in the diff, forgot to commit/push that bit?

What do you mean? I am not following, sorry

**Comment by pnuu** [comment]
> > I don't see `utcnow` in the diff, forgot to commit/push that bit?
> 
> What do you mean? I am not following, sorry

Above @djhoese said

> Oh I also made pytest more strict and removed usage of datetime utcnow.

I don't see anything related to `utcnow` in the PR diff, so I was wondering whether something is still missing.

**Comment by djhoese** [comment]
Ah yep. I'm just waking up and won't be in front of the computer for a while (kid's doctor appointment). I'll see if I can push it while getting kids ready for the day.

File: pyorbital/tlefile.py [hunk 1]
```diff
@@ -617,6 +611,9 @@ def table_exists(db, name):
     return db.execute(query, (name,)).fetchone() is not None  # nosec
 
 
+def _utcnow():
+    return dt.datetime.now(tz=dt.timezone.utc).replace(tzinfo=None)
```
Comment (mraspaud) [inline pyorbital/tlefile.py:615]: why do we want to remove the timezone info again?

Comment (djhoese) [reply pyorbital/tlefile.py:615]: To make them naive. Keeping the timezone would require a lot more work I assume. For example, sanitizing user inputs.

Comment (mraspaud) [reply pyorbital/tlefile.py:615]: ok

**Review by mraspaud — APPROVED** [review]
LGTM! I suppose v1.9 is the next release then, right?

**Comment by mraspaud** [comment]
oh, looks like tests are failing...

**Comment by djhoese** [comment]
Since this requires a 1.9 release I'll let others (@adybbroe ?) answer whether or not this should be merged and released in 1.9 next or if there should be a 1.8.x release first.

File: pyorbital/tlefile.py (hunk 1 above)
Comment (pnuu) [reply pyorbital/tlefile.py:625]: If the datetimes are timezone-unaware, they can't be compared with tz-aware datetimes. I thought our goal was to make everything tz-aware across the Pytroll ecosystem.

Comment (pnuu) [reply pyorbital/tlefile.py:625]: And I think makin all of the datetimes `tzinfo=None` is even more work than make them use UTC. The `None` case would mean addinitional TZ conversion for pretty much everywhere where we declare the times as UTC.

File: pyorbital/tlefile.py [hunk 2]
```diff
@@ -617,6 +621,9 @@ def table_exists(db, name):
...
```
Comment (pnuu) [inline pyorbital/tlefile.py:625]: I'll just add the official "request changes" for this bit.

**Review by pnuu — CHANGES_REQUESTED** [review]

File: pyorbital/tlefile.py (hunk 1 above)
Comment (djhoese) [reply pyorbital/tlefile.py:625]: I don't disagree that we should/could move toward UTC-aware times everywhere, but that wasn't my goal with my changes in this PR. I was mostly concerned about the warnings generated while running tests. I understand about 2% of pyorbital and the TLE portions, on a code level, are not part of that understanding. I should not be the person who does that conversion/translation in pyorbital. Maybe I'm overestimating how much datetimes are involved. If so I can take a stab at it, but I'm still not sure this is the PR for it.

**Review by pnuu — APPROVED** [review]
LGTM.

I merged `main` and resolved the conflicts, should be ready to merge now.

**Comment by pnuu** [comment]
Version 1.9.0 is now released, so I'll merge this.

**Comment by djhoese** [comment]
This was meant to be released as part of 1.9.0 based on the warning messages.

**Comment by pnuu** [comment]
Damn, I read this the wrong way around :see_no_evil: 

> Since this requires a 1.9 release I'll let others (@adybbroe ?) answer whether or not this should be merged and released in 1.9 next or if there should be a 1.8.x release first.

**Comment by djhoese** [comment]
Maybe a 1.9.1 release then?

**Comment by pnuu** [comment]
Will do, now that the release deployment was fix in #172 :grin:

**Pull request merged**